kuma/payments/jinja2/payments/management.html

```diff
@@ -0,0 +1,128 @@
+                <table class="bordered-info">
+                    <thead>
+                        <tr>
+                            <th>{% include 'includes/icons/general/money-bill-alt.svg' %} {{_('Amount')}}</th>
+                            <th>{% include 'includes/icons/general/credit-card.svg' %} {{_('Card Number')}}</th>
+                            <th>{% include 'includes/icons/general/credit-card.svg' %} {{_('Expiry')}}</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td>${{ stripe_plan_amount }}</td>
+                            <td>**** **** **** {{ stripe_card_last4 }}</td>
+                            <td>02/19</td>
+                        </tr>
+                    </tbody>
+                </table>
+                {% if not cancel_request %}
+                    <form id="id_stripe_management_form" method="post">
+                        {% csrf_token %}
+                        <button class="button negative" type="button" toggle-confirmation>{{_('Cancel your subscription')}}</button>
+                        <div id="delete-confirmation" class="hidden" aria-hidden="true">
+                            <div class="confirmation column-8 alert danger">
+                                <h5>{{_('Are you sure?')}}</h5>
+                                <p>
+                                    {{_('You will have to set up a new subscription if you wish to resume making payments to MDN.')}}
+                                </p>
+                                <div class="button-group">
+                                    <button class="button" type="button" toggle-confirmation>{{_('Cancel')}}</button>
+                                    <button id="id_stripe_cancel_subscription" name="stripe_cancel_subscription" class="button negative" type="submit">{{_('Confirm')}}</button>
+                                </div>
+                            </div>
+                        </div>
+                    </form>
+                {% endif %}
+            </section>
+        {% else %}
+            <section>
+                <h3>{{_('Monthly Subscription')}}</h3>
+                <div class="bordered-info">
+                    <span>
+                        {% trans url=url('recurring_payment_initial') %}
+                            You have no active susbcriptions.
+                            Why not <a href="{{ url }}">set one up?</a>
+                        {% endtrans %}
+                    </span>
+                </div>
+            </section>
+        {% endif %}
+        {% if cancel_request %}
+            {% if cancel_success %}
+                <div class="confirmation column-8 alert success">
+                    <b>{{_('Your monthly subscription has been successfully cancelled')}}</b>
+                </div>
+            {% else %}
+                <div class="confirmation column-8 alert success">
```

Comment:
This uses the `success` class but it is an error state?

Reply:
Yep! My bad for blindly duplicating code.  Addressed.